scss/civicrm/dashboard/pages/_edit-dashboard.scss

```diff
@@ -0,0 +1,92 @@
+#help {
```

Comment:
I think you should wrap this in `#configure-dashlet`, the wrapping shouldn't be outside of the partial